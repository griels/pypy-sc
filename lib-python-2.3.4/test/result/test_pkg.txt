command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_pkg.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Wed Apr 27 22:27:08 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_pkg.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
========================== closed ==========================
execution time: 29.933478117 seconds
exit status: 0
