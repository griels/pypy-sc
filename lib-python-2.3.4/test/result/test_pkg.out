test_pkg
running test t1
mkdir /tmp/tmpyjFK1y/t1
write /tmp/tmpyjFK1y/t1/__init__.py
sys.path = ['/tmp/tmpyjFK1y', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
rmdir /tmp/tmpyjFK1y/t1
rmdir /tmp/tmpyjFK1y
running test t2
mkdir /tmp/tmpO81fq5/t2
write /tmp/tmpO81fq5/t2/__init__.py
mkdir /tmp/tmpO81fq5/t2/sub
write /tmp/tmpO81fq5/t2/sub/__init__.py
mkdir /tmp/tmpO81fq5/t2/sub/subsub
write /tmp/tmpO81fq5/t2/sub/subsub/__init__.py
sys.path = ['/tmp/tmpO81fq5', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
t2 loading
doc for t2
t2.sub.subsub loading
t2 t2.sub t2.sub.subsub
['sub', 't2']
t2.sub t2.sub.subsub
t2.sub.subsub
['spam', 'sub', 'subsub', 't2']
t2 t2.sub t2.sub.subsub
['spam', 'sub', 'subsub', 't2']
rmdir /tmp/tmpO81fq5/t2/sub/subsub
rmdir /tmp/tmpO81fq5/t2/sub
rmdir /tmp/tmpO81fq5/t2
rmdir /tmp/tmpO81fq5
running test t3
mkdir /tmp/tmpZAAbFD/t3
write /tmp/tmpZAAbFD/t3/__init__.py
mkdir /tmp/tmpZAAbFD/t3/sub
write /tmp/tmpZAAbFD/t3/sub/__init__.py
mkdir /tmp/tmpZAAbFD/t3/sub/subsub
write /tmp/tmpZAAbFD/t3/sub/subsub/__init__.py
sys.path = ['/tmp/tmpZAAbFD', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
t3 loading
t3.sub.subsub loading
t3 t3.sub t3.sub.subsub
t3 loading
Traceback (most recent call last):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_pkg.py", line 66, in runtest
    execfile(fname, globals(), {})
  File "/tmp/tmpZLWDPt", line 5, in ?
    reload(t3.sub)
AttributeError: 'module' object has no attribute 'sub'
rmdir /tmp/tmpZAAbFD/t3/sub/subsub
rmdir /tmp/tmpZAAbFD/t3/sub
rmdir /tmp/tmpZAAbFD/t3
rmdir /tmp/tmpZAAbFD
running test t4
write /tmp/tmpkf50O2/t4.py
mkdir /tmp/tmpkf50O2/t4
write /tmp/tmpkf50O2/t4/__init__.py
write /tmp/tmpkf50O2/t4/sub.py
mkdir /tmp/tmpkf50O2/t4/sub
write /tmp/tmpkf50O2/t4/sub/__init__.py
write /tmp/tmpkf50O2/t4/sub/subsub.py
mkdir /tmp/tmpkf50O2/t4/sub/subsub
write /tmp/tmpkf50O2/t4/sub/subsub/__init__.py
sys.path = ['/tmp/tmpkf50O2', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
t4 loading
t4.sub.subsub loading
t4.sub.subsub.spam = 1
rmdir /tmp/tmpkf50O2/t4/sub/subsub
rmdir /tmp/tmpkf50O2/t4/sub
rmdir /tmp/tmpkf50O2/t4
rmdir /tmp/tmpkf50O2
running test t5
mkdir /tmp/tmpB4z795/t5
write /tmp/tmpB4z795/t5/__init__.py
write /tmp/tmpB4z795/t5/string.py
write /tmp/tmpB4z795/t5/foo.py
sys.path = ['/tmp/tmpB4z795', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
t5.foo loading
t5.string loading
1
['foo', 'string', 't5']
['__file__', '__name__', '__path__', 'foo', 'string', 't5']
['__file__', '__name__', 'string']
['__file__', '__name__', 'spam']
rmdir /tmp/tmpB4z795/t5
rmdir /tmp/tmpB4z795
running test t6
mkdir /tmp/tmpKIkd4b/t6
write /tmp/tmpKIkd4b/t6/__init__.py
write /tmp/tmpKIkd4b/t6/spam.py
write /tmp/tmpKIkd4b/t6/ham.py
write /tmp/tmpKIkd4b/t6/eggs.py
sys.path = ['/tmp/tmpKIkd4b', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
['__all__', '__file__', '__name__', '__path__']
Traceback (most recent call last):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_pkg.py", line 66, in runtest
    execfile(fname, globals(), {})
  File "/tmp/tmpL8EK10", line 4, in ?
    from t6 import *
AttributeError: 'module' object has no attribute 'spam'
rmdir /tmp/tmpKIkd4b/t6
rmdir /tmp/tmpKIkd4b
running test t7
write /tmp/tmpZfUKRE/t7.py
mkdir /tmp/tmpZfUKRE/t7
write /tmp/tmpZfUKRE/t7/__init__.py
write /tmp/tmpZfUKRE/t7/sub.py
mkdir /tmp/tmpZfUKRE/t7/sub
write /tmp/tmpZfUKRE/t7/sub/__init__.py
write /tmp/tmpZfUKRE/t7/sub/subsub.py
mkdir /tmp/tmpZfUKRE/t7/sub/subsub
write /tmp/tmpZfUKRE/t7/sub/subsub/__init__.py
sys.path = ['/tmp/tmpZfUKRE', '', '/u/pedronis/PyPy/dist/pypy/lib', '/u/pedronis/PyPy/dist/lib-python-2.3.4', '/usr/lib/python23.zip', '/usr/lib/python2.3', '/usr/lib/python2.3/plat-linux2', '/usr/lib/python2.3/lib-tk', '/usr/lib/python2.3/lib-dynload', '/usr/local/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages', '/usr/lib/python2.3/site-packages/Numeric', '/usr/lib/python2.3/site-packages/PIL', '/usr/lib/site-python']
t7 loading
['__file__', '__name__', '__path__']
['__file__', '__name__', '__path__']
t7.sub.subsub loading
['__file__', '__name__', '__path__', 'spam']
t7.sub.subsub.spam = 1
rmdir /tmp/tmpZfUKRE/t7/sub/subsub
rmdir /tmp/tmpZfUKRE/t7/sub
rmdir /tmp/tmpZfUKRE/t7
rmdir /tmp/tmpZfUKRE
